tfhe/src/high_level_api/integers/tests_signed.rs

```diff
@@ -0,0 +1,650 @@
+use crate::integer::I256;
```

Comment:
I believe HL API tests are not run 🤔 could we add a Makefile target ?

(could not find them in the integer tests run)

Reply:
it's done in the fast tests it looks like my bad